src/binxml/assemble.rs

```diff
@@ -228,39 +259,48 @@ fn expand_borrowed_template<'a>(
     // Here we can always use refs, since even if the definition is owned by the template,
     // we do not own it.
     for token in template.definition.as_ref().tokens.iter() {
-        if let BinXMLDeserializedTokens::Substitution(ref substitution_descriptor) = token
-        {
+        if let BinXMLDeserializedTokens::Substitution(ref substitution_descriptor) = token {
             if substitution_descriptor.ignore {
                 continue;
             } else {
-                let value = &template.substitution_array[substitution_descriptor.substitution_index as usize];
+                let value = &template.substitution_array
```

Comment:
IMO this should be considered a bug in `rustfmt`, this is just terrible 😞

Reply:
sorry about that :( we could turn it off manually if it's that bad